cmd/manager/main.go

```diff
@@ -93,9 +90,8 @@ func main() {
 	// watch for CatalogSources in targetNamespaces being deleted and recreate
 	// them.
 	mgr, err := manager.New(cfg, manager.Options{
-		Namespace:          namespace,
-		MapperProvider:     restmapper.NewDynamicRESTMapper,
-		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
```

Comment:
Can you explain why this is removed? I assume it is to prevent us from reporting sdk metrics, but I want to confirm.

Reply:
Yes, `MetricsBindAddress` would run the command to register default metrics with at the port/path provided by SDK. Since we are adding our own custom metrics to it and registering them, this needs to be removed. Or else, we would be registering metrics to the same port twice, which will result in an error.